modules/Subsetting_Data_in_R/Subsetting_Data_in_R.Rmd

```diff
@@ -695,25 +646,23 @@ https://media.giphy.com/media/5b5OU7aUekfdSAER5I/giphy.gif
 
 ## Lab Part 2
 
-🏠 [Class Website](https://daseh.org/)    
+🏠 [Class Website](https://daseh.org)    
 💻 [Lab](https://daseh.org/modules/Subsetting_Data_in_R/lab/Subsetting_Data_in_R_Lab.Rmd)
 
 ## Get the data
 
 ```{r}
-#install.packages(OCSdata)
-#library(OCSdata)
-#OCSdata::load_imported_data("ocs-bp-opioid-rural-urban")
+#read_csv("https://daseh.org/data/Colorado_ER_heat_visits_by_county_gender.csv")
```

Comment:
library(dasehr)
CO_heat_ER_bygender

?

Reply:
sure I will add it